src/routes/users.test.js

```diff
@@ -0,0 +1,33 @@
+const request = require('../test/supertest');
+const User = require('../models/user');
+const emails = require('../emails');
+
+describe('users route tests', () => {
+  describe('POST users/send-token', () => {
+    const sendTokenUrl = '/users/send-token';
+    const userEmail = 'test1@example.org';
+    const sendTokenEmailSpy = jest.spyOn(emails, 'sendToken');
+
+    afterEach(async () => {
```

Comment:
it's the same at the end but this should be `beforeEach`, because you care of having a clear mock when you start the test, not after doing it

```suggestion
    beforeEach(async () => {
```